tony-core/src/main/resources/tony-default.xml

```diff
@@ -164,6 +164,12 @@
     <value>0</value>
   </property>
 
+  <property>
+    <description>Resources that will be localized to the container that executes the job. Delimited by comma.</description>
```

Comment:
Same here, can we just say copied to the container home dir instead of localized?

Reply:
yah, will update. For the issue below:
```
we shouldn't need to specify the hdfs uri right? Haven't tested it, but since we're calling fs.copyToLocalFile it should assume src is on hdfs and dst is local?
```
I want to unify this with `src_dir` flag. Basically the resource can be either local or in remote fs.

Reply:
There seems to be bug in github that I can't reply the comment below for some reason.. thus replying here.